src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts

```diff
@@ -570,11 +578,17 @@ class ZWaveJSConfigDashboard extends SubscribeMixin(LitElement) {
   }
 
   private async _addNodeClicked() {
```

Comment:
Is it possible to click the button while the dialog is open? I get that we restrict it on the server message, but it feels kinda "wobbly" the way we do it, so I would rather not restrict the user interaction if not needed...